capstone/src/test/java/com/google/univiz/api/resource/SearchResourceImplTest.java

```diff
@@ -58,4 +63,16 @@ public void testNullCollegeName() throws Exception {
     List<SearchData> ret = search.getSearchSuggestions(collegeName);
     assertThat(ret).isEmpty();
   }
+
+  @Test
+  public void testFilteringIncorrectSuggestions() throws Exception {
+    String collegeName = "Stan";
+    when(mockSuggestionApi.getCollegeSuggestions(collegeName))
+        .thenReturn(MULTIPLE_SUGGESTION_RESPONSE);
+
+    List<SearchData> ret = search.getSearchSuggestions(collegeName);
+    CollegeId collegeId = CollegeId.create(STANFORD_SUGGESTION_DATA.id());
+    SearchData expected = SearchData.create("Stanford University", collegeId);
+    assertThat(ret).containsExactly(expected);
+  }
```

Comment:
```suggestion
  }
  
  @Test
  public void testIoException() {}
```
i would add a test to make sure it accepts the exception. (chris would probably ask for a test to make sure they throw exceptions haha). The `MapsResourceImplTest.java` class in #87 has an example of it.

Reply:
done!